lighthouse-core/lib/dependency-graph/simulator/simulator.js

```diff
@@ -320,8 +328,10 @@ class Estimator {
     const connection = this._connections.get(node.record.connectionId);
     const calculation = connection.simulateDownloadUntil(
       node.record.transferSize - timingData.bytesDownloaded,
-      timingData.timeElapsed,
-      timePeriodLength - timingData.timeElapsedOvershoot
+      {
```

Comment:
wait, why is this moving to an options object? There's only three and having them named doesn't seem better in this case...

Reply:
I feel ya @brendankenny, but @paulirish was persistent

executive summary: it's easier for the caller to understand what the params are